compiler/rustc_data_structures/src/transitive_relation.rs

```diff
@@ -1,5 +1,6 @@
 use crate::frozen::Frozen;
 use crate::fx::FxIndexSet;
+use rustc_hash::FxHashSet;
```

Comment:
Nit: `FxHashSet` is typically imported from `rustc_data_structures` rather than from `rustc_hash`.